modules/ROOT/pages/split-aggregate-scope.adoc

```diff
@@ -17,6 +17,7 @@ The Split Aggregate scope can be configured through the following fields:
 [%header,cols="1,3"]
 |===
 |Attribute |Description
+|Collection Expression (`collection`) | An expression to that returns a Java collection, object array, map, or DOM nodes. By default, the payload is taken as the collection to split.
```

Comment:
`An expression that returns`? I also think we should recheck this and compare to the foreach docs. With DW you don't need for it to be Java for example, it could be a JSON array.

Reply:
I literally copied this from the foreach doc, https://docs.mulesoft.com/mule-runtime/4.1/for-each-scope-xml-reference. I will update both.